tools/myst/enc/enc.c

```diff
@@ -492,6 +493,7 @@ static long _enter(void* arg_)
     bool perf = false;
     bool report_native_tids = false;
     size_t max_affinity_cpus = options ? options->max_affinity_cpus : 0;
+    size_t main_stack_size = options ? options->main_stack_size : 0;
```

Comment:
In case option is not provided, shouldn't main_stack_size be set to implicit default of MYST_PROCESS_INIT_STACK_SIZE?

Reply:
Correct. Actually the default value is set at line 767-768. In this function, the ultimate value `_kargs.main_stack_size` is determined in three steps:

1. In line 496, check commandline option value. If it is not provided, then set 0.
2. In line 631-634, check config.json parameter value. If it is set, override commandline option.
3. In line 767-768, if main_stack_size is set by neither commandline option nor config.json, then assign it with default value MYST_PROCESS_INIT_STACK_SIZE.

The reason why I didn't set main_stack_size to default value here in step 1, but in step 3, is in case user specify main_stack_size=0 in config.json by mistake. Step 3 will change that value to MYST_PROCESS_INIT_STACK_SIZE.